libfwupd/fwupd-enums.h

```diff
@@ -553,6 +553,15 @@ typedef enum {
  * Since: 1.9.3
  */
 #define FWUPD_DEVICE_FLAG_ONLY_EXPLICIT_UPDATES (1llu << 51)
+/**
+ * FWUPD_DEVICE_FLAG_GENERIC_GUID:
```

Comment:
Don't like this name for describing this. It's really preventing the use of matching a guid and forcing other requirements. In those cases aren't we better off having a flag that enforces requirements are set? And/or removing the "generic" guids from problematic devices?

Reply:
I agree the flag name is crap too. Maybe `FWUPD_DEVICE_FLAG_ENFORCE_REQUIRES` or something like that?

Reply:
Yeah that's a better name.

And yeah I was thinking an internal flag instead of an exported flag if it's really just for one plugin. But if it's gonna be used over and over may as well go into global flags

Reply:
I've moved it to internal flags -- you're right that it's not needed in clients.